src/Ssl/CertificateRequest.php

```diff
@@ -24,19 +24,37 @@ class CertificateRequest
     /** @var KeyPair */
     private $keyPair;
 
-    public function __construct(DistinguishedName $distinguishedName, KeyPair $keyPair)
+    /** @var CsrPem */
```

Comment:
Isn't it a string?

Reply:
csrPem variable? Yes, csrPem is a string (base64 encoded PKCS#10).

Reply:
Then the php doc is wrong :)

Reply:
Not sure if we are talking about the same thing :/ php doc?